core/src/test/scala/sttp/client4/testing/HttpTest.scala

```diff
@@ -57,6 +57,7 @@ trait HttpTest[F[_]]
   protected def supportsCancellation = true
   protected def supportsAutoDecompressionDisabling = true
   protected def supportsDeflateWrapperChecking = true
+  protected def supportsEmptyContentEncoding = false
```

Comment:
eee ... shouldn't this be `true` by default? As in here https://github.com/softwaremill/sttp/blob/v3/core/src/test/scala/sttp/client3/testing/HttpTest.scala#L60

Reply:
Hmm seems so.